drivers/net/phy/micrel.c

```diff
@@ -577,6 +577,13 @@ static int ksz8041_fiber_mode(struct phy_device *phydev)
 	return of_property_read_bool(of_node, "micrel,fiber-mode");
 }
 
+static int ksz8081_is_enable_broadcast_address(struct phy_device *phydev)
```

Comment:
Lets rename to kszphy_is_broadcast_address_allowed() since this feature is not exclusively specific to the KSZ8081 device.